client/app/dbaas/logs/detail/home/logs-home.html

```diff
@@ -65,7 +65,7 @@
                             </button>
                         </oui-tile-description>
                     </oui-tile-definition>
-                    <oui-tile-definition>
+                    <oui-tile-definition data-term="{{ctrl.accountDetails.data.graylogEntryPoint}}">
```

Comment:
Feedback from QA team:

Could you precise a more user friendly information?
Instead of `"graX.logs.ovh.com"` something like `"Accès aux logs"`?